pyproject.toml

```diff
@@ -1,3 +1,84 @@
+[tool.poetry]
+name = "prospector"
+version = "1.5.0.dev2"
+description = ""
+authors = ["Carl Crowder <git@carlcrowder.com>"]
+maintainers = ["Carl Crowder <git@carlcrowder.com>",
+               "Carlos Coelho <carlospecter@gmail.com>"]
```

Comment:
@Pierre-Sassoulas please add yourself to this list if you want to! Also to `CONTRIBUTERS.rst`.

Reply:
```suggestion
               "Carlos Coelho <carlospecter@gmail.com>",
               "Pierre Sassoulas <pierre.sassoulas@gmail.com>"]
```

Reply:
:+1: